srcpkgs/polybar/template

```diff
@@ -23,9 +23,9 @@ makedepends="cairo-devel xcb-util-image-devel xcb-util-wm-devel xcb-util-xrm-dev
 short_desc="Fast and easy-to-use status bar"
 maintainer="Michael Carlberg <c@rlberg.se>"
 license="MIT"
-homepage="https://github.com/jaagr/polybar"
-distfiles="https://github.com/jaagr/polybar/releases/download/${version}/polybar-${version}.tar.gz"
-checksum=133af4e8b29f426595ad3b773948eee27275230887844473853e7940c7959c2b
+homepage="https://github.com/polybar/polybar"
+distfiles="${homepage}/releases/download/${version}/${pkgname}-${version}.tar.gz"
```

Comment:
```suggestion
distfiles="${homepage}/releases/download/${version}/polybar-${version}.tar.gz"
```

Neither is very probable, but it's more likely for our package to change name than what upstream uses, so I'd rather keep it typed out explicitly.